Readme.md

```diff
@@ -62,6 +64,28 @@ Any job related work may be clocked in your timesheet. This includes, but is def
 Just make sure you don't work for free and that your timesheet accurately reflects the hours you worked. Having some tractability here
 is helpful, but not strictly necessary.
 
+## When can I clock hours
+
+The department enforces a variety of policies in regards to when hours can and cannot be clocked.
+Most importantly, *HOURS MUST NOW BE CLOCKED DAILY*.
+You must, as part of their paid duties, fill out their time sheet daily with the in and out times they worked.
```

Comment:
remove "their" (grammar) same for later in the sentence.  Or change to "Students must" to reflect language of the other notes